Dockerfile.src

```diff
@@ -0,0 +1,59 @@
+FROM {{ config.docker.registry }}/{{ config.docker.from }}
```

Comment:
I guess, this will not cover upstream / downstream specification.
There should be {{ spec.variant.image_name }}, right?

Reply:
good point!

Reply:
hold on, this is base image, so it's correct, you are talking about the image this dockerfile builds